### PR TITLE
create result-report and result-report-item entity

### PR DESCRIPTION
**Pull request by YuriiZaripa**

### CODE REVIEW

File: src/main/java/com/parsing/model/ResultReportItem.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Entity
+@Setter
+@Getter
+@ToString(exclude = "resultReport")
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "result_report")
```
Comment (OleksandrOpanasiuk) [inline src/main/java/com/parsing/model/ResultReportItem.java]: Maybe name = result_report_item?

Comment (YuriiZaripa) [reply src/main/java/com/parsing/model/ResultReportItem.java]: thank you!

File: src/main/java/com/parsing/model/ResultReport.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package com.parsing.model;
+
+
```
Comment (OleksandrOpanasiuk) [inline src/main/java/com/parsing/model/ResultReport.java]: one empty string.

File: src/main/java/com/parsing/model/ResultReport.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Table;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.CascadeType;
+import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+
```
Comment (OleksandrOpanasiuk) [inline src/main/java/com/parsing/model/ResultReport.java:14]: maybe delete empty strings.

File: src/main/java/com/parsing/model/ResultReportItem.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import lombok.Setter;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Entity
+@Setter
+@Getter
```
Comment (OleksandrOpanasiuk) [inline src/main/java/com/parsing/model/ResultReportItem.java:21]: You are using @Data and @Getter & @Setter. Let's use only one.

Comment (FedosovMax) [reply src/main/java/com/parsing/model/ResultReportItem.java:22]: Can't see @Data

**Review by FedosovMax — APPROVED** [review]

**Review by FedosovMax — APPROVED** [review]

**Pull request merged**